### ISSUE TITLE
Release 3.3.3

### ISSUE THREAD

**Issue by xzyfer** [sass/libsass#1808]
This issues tracks the 3.3.3 release. Subscribe to this if you are waiting for a fix in that version.

- [x] Applaud @mgreter :clap: :clap: :clap: :clap: 
- [x] Make sure we don't break eyeglass /cc @chriseppstein 
- [x] Make sure we've fixed sassdash /cc @davidkpiano
- [x] Release notes
- [x] Release SassC
- [x] Release LibSass

**Comment by xzyfer** [comment]
Added a release todo list /cc @mgreter @saper

**Comment by tehOPEologist** [comment]
/applauds @mgreter :clap: :clap: :clap: :clap:  ^_^

**Comment by xzyfer** [comment]
@mgreter @saper I've confirmed both eyeglass and sassdash have fully passing test suites with node-sass compiled against @a559139.

I'm happy to call this release ready and start cutting release notes. Objections?

**Comment by mgreter** [comment]
Not really, 3.3.4 might follow shortly after (need to wait for 3.4 before all API changes are ready).

**Comment by xzyfer** [comment]
Tying up some loose ends on the node-sass side of things. I'll tag the release in the next 48hrs.

**Comment by xzyfer** [comment]
@mgreter we need to feature freeze before I can cut the release. Let me know when you're done.

**Comment by mgreter** [comment]
@xzyfer I thought you would first prepare the notes? Waiting for these, then we can freeze.

**Comment by mgreter** [comment]
Went ahead and created them via:
```perl
use strict;
use warnings;

use pQuery;

my $loop = 1;
my $page = 0;
my $count = 0;
my $tag = "3.3.3";

while ($loop) {

  $loop = 0;
  $page ++;

  pQuery("http://github.com/sass/libsass/issues?page=$page&q=milestone%3A$tag+is%3Aclosed")
    ->find('LI[id^="issue_"]')
    ->each(sub {
        $loop = 1;
        $count ++;
        my $i = shift;
        my $doc = pQuery($_);
        my $title = $doc->find('.issue-title-link')->text;
        my $user = $doc->find('.table-list-cell-avatar IMG')->attr('alt');
        my $id = $doc->attr('id');
        $id = $1 if $id =~ m/issue_(\d+)/;
        print "- ", $title, " (", ($user ? $user : "XXX"), ", #$id)\n";

    });

}
```

**Comment by mgreter** [comment]
@xzyfer can you release and test sassc. I will not have time this weekendè

**Comment by xzyfer** [comment]
Will. I need to spend some time getting my node sass releases working again also.

**Comment by mgreter** [comment]
Any progress here?

**Comment by xzyfer** [comment]
Released [SassC 3.3.3](https://github.com/sass/sassc/releases/tag/3.3.3)

**Comment by saper** [comment]
:tada:
